README.md

```diff
@@ -36,6 +36,10 @@ The following types of data are being considered:
  | URL         | Url with more information on the image|
  | Summary     | Short Description of the image|
  | Description | Detailed description of the image|
+ | VcsType    | Where the image source can be obtained from a version control system, which software is used|
```

Comment:
Perhaps: "The type of version control used by the application packaged in this container, generally one of git, hg, svn, cvs, bzr"